test-bench-utils/lib/sinks/xxe.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
-
-const libxmljs = require('libxmljs');
+let libxmljs;
```

Comment:
~probably worth throwing a comment as to why we expect this to be non-null in some cases.~ nm see it below